doc/security/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "OpenTitan Security Introduction"
+---
+
+## Overview
+
+OpenTitan’s mission is to create a trustworthy, vendor-agnostic open source silicon Root of Trust (RoT) widely adopted across the industry.
+The project aims to accomplish this by implementing strong logical security integrity guarantees in the RTL and firmware components, and by providing a comprehensive certification plan for physical implementations.
```

Comment:
i wonder if 'HW' would be better here than 'RTL'.

Reply:
I just wanted to make a distinction between physical implementations versus RTL. wdyt?

Reply:
i see what you mean..but i worry RTL is a bit overly specific because it seems to leave out things like analog design, pads etc.  Unless those you consider physical implementation?

Reply:
"silicon"?

Reply:
SGTM. Updated this with a wording suggested by @domrizz0. PTAL.